src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -108,27 +83,23 @@ public String getId() {
 
   @Nullable
   @Override
-  public Runnable enableSearch(String option) {
-    //TODO(pq): enable search
```

Comment:
I'd like to be sure we implement search and would rather leave a TODO than delete it.  Unless you think we shouldn't?